pvlib/modelchain.py

```diff
@@ -408,6 +421,41 @@ def sapm(self):
 
         return self
 
+    def desoto(self):
+        (photocurrent, saturation_current, resistance_series,
+         resistance_shunt, nNsVth) = (
+            self.system.calcparams_desoto(self.effective_irradiance,
+                                          self.temps['temp_cell']))
+
+        self.diode_params = (photocurrent, saturation_current, resistance_series,
+                       resistance_shunt, nNsVth)
```

Comment:
Should read as:

```
        self.diode_params = (photocurrent, saturation_current,
                             resistance_series, resistance_shunt, nNsVth)
```

same in ``pvsyst`` below. Not sure why the git diff/flake8 command doesn't pick up on this. Maybe time for another github/CI integration tool.

Reply:
Fixed here and in `pvsyst`